torch/_dynamo/config_utils.py

```diff
@@ -35,7 +36,9 @@ def visit(source, dest, prefix):
                 if dest is module:
                     delattr(module, key)
             elif isinstance(value, type):
-                assert value.__module__ == module.__name__
+                if issubclass(value, Enum):
+                    continue
```

Comment:
Seems questionable

Reply:
This whole file is questionable, but it works?

Reply:
That's not good enough. Why does Enum need to get special cased to continue, whereas every other type falls through below? Maybe the handling should be somewhere else?